apps/cli/src/node/docker-compose-dev.yaml

```diff
@@ -91,30 +91,78 @@ services:
         depends_on:
             validator:
                 condition: service_healthy
+            traefik-config-generator:
+                condition: service_completed_successfully
         healthcheck:
-            test: ["CMD", "traefik", "healthcheck"]
+            test: ["CMD", "traefik", "healthcheck", "--ping"]
             interval: 5s
             timeout: 1s
             retries: 5
+        command:
+            [
+                "--ping=true",
+                "--entryPoints.web.address=:8088",
+                "--entryPoints.traefik.address=:8080",
+                "--metrics.prometheus=true",
+                "--metrics.prometheus.addServicesLabels=true",
+                "--providers.file.directory=/etc/traefik/conf.d",
+                "--providers.file.watch=true"
+            ]
         ports:
-            - 8080:8080
+            - 8080:8088
+            - 8088:8080
```

Comment:
Is this reverted? Can we not expose the management port?

Reply:
I'll remove the `:8088` port.